zealot/src/zng/types/record.ts

```diff
@@ -49,7 +53,7 @@ export class Record implements ZngClass<Type[] | null> {
   at(index: number) {
     const col = this.type[index]
     if (!col) throw new Error(`No column at index: ${index}`)
-    const val = this.value[index]
+    const val = this.isSet() ? this.value![index] : null
```

Comment:
@jameskerr: The isSet type guard resulted in `Error: src/js/electron/menu/actions/searchActions.ts(207,31): error TS2352: Conversion of type 'Type[]' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.`, so I tried these non-null assertion. Not sure if they're kosher but they seem to do the job.